dashboard/app/controllers/levels_controller.rb

```diff
@@ -563,30 +573,26 @@ def extra_links
         links["Template Level"] = [
           {text: project_template_level_name, url: level_path(project_template_level)}
         ]
-        template_level_edit_link =
-          if can_edit_level
-            {text: 'Edit', url: edit_level_path(project_template_level)}
-          else
-            {text: '(Cannot edit)', url: ''}
-          end
+        template_level_edit_link = can_edit_level ?
+          {text: 'Edit', url: edit_level_path(project_template_level)} :
+          {text: '(Cannot edit)', url: ''}
         links["Template Level"] << template_level_edit_link
       end
-
-    elsif @script_level
+    elsif script_level
```

Comment:
Chatted with Molly about this, and we don't think this elsif condition ever worked (ie, I never found @ script_level or @ extra_params to be defined when looking at a script level URL). Looks like a possible copy/past from here:

https://github.com/code-dot-org/code-dot-org/blob/0139c565b648857fa3bce063ff9693cb761a35ad/dashboard/app/views/levels/_admin.html.haml#L75